buildSrc/build.gradle.kts

```diff
@@ -29,8 +29,8 @@ dependencies {
   implementation(gradleApi())
   implementation(localGroovy())
 
-  implementation("io.opentelemetry.instrumentation.muzzle-generation:io.opentelemetry.instrumentation.muzzle-generation.gradle.plugin:0.7.0")
-  implementation("io.opentelemetry.instrumentation.muzzle-check:io.opentelemetry.instrumentation.muzzle-check.gradle.plugin:0.7.0")
+  implementation("io.opentelemetry.instrumentation.muzzle-generation:io.opentelemetry.instrumentation.muzzle-generation.gradle.plugin:0.8.0-SNAPSHOT")
+  implementation("io.opentelemetry.instrumentation.muzzle-check:io.opentelemetry.instrumentation.muzzle-check.gradle.plugin:0.8.0-SNAPSHOT")
```

Comment:
is this intentional?

Reply:
Somewhat - I wondered whether I should release a 0.8.0 plugins version that depends on 1.6.0-SNAPSHOT or depend on 0.8.0-SNAPSHOT here, I thought the latter might be a bit simpler. I'm not sure what's the correct choice here though.

Reply:
I think there are no good choices, so let's go with this